src/YAML.jl

```diff
@@ -6,6 +6,7 @@ import Base: start, next, done, isempty, length, show
 import Codecs
 using Compat
 import Compat: String
+import Compat: Iterators
```

Comment:
this needs 0.18 https://github.com/JuliaLang/Compat.jl/commit/a53f1023da52c5098e5ddb70ef49379fcfee1c87